src/pages/development/components/message-queues/migration.md

```diff
@@ -8,6 +8,207 @@ keywords:
+
+### Key Changes in 2.4.6+
+
+- **ActiveMQ Artemis (STOMP) Support**: New message broker option alongside RabbitMQ (introduced in 2.4.9, backported to 2.4.6, 2.4.7, 2.4.8)
+- **Extended Dynamic Connection Detection**: Existing dynamic connection detection now supports STOMP in addition to AMQP
+- **Enhanced SSL Configuration**: Improved SSL options for both brokers
+- **Multiple Named Connections**: Enhanced support for configuring multiple broker connections
+
+### Configuration File Changes
+
+The following table shows the key differences between 2.4.5 and 2.4.6+ configurations:
+
+| Configuration Area | 2.4.5 | 2.4.6+ |
+| ------------------ | ----- | ------ |
+| `env.php` default_connection | Optional when single broker configured | Optional when single broker configured; required when multiple brokers (can be `'amqp'`, `'stomp'`, or `'db'`) |
+| `env.php` broker configuration | RabbitMQ (AMQP) only | RabbitMQ (AMQP) and/or ActiveMQ Artemis (STOMP) |
+| `queue_consumer.xml` connection attribute | Optional (auto-detected from env.php) | Optional (auto-detected, now includes STOMP in 2.4.6+) |
+| `queue_publisher.xml` connection element | Optional for AMQP, supports multiple connections | Optional for AMQP/STOMP, enhanced multiple connections |
+| `queue_topology.xml` connection attribute | Optional (auto-detected from env.php) | Optional (auto-detected, now includes STOMP in 2.4.6+) |
+
+### Update `env.php` Configuration
+
+**For RabbitMQ (AMQP) - No Changes Required:**
+```php
+'queue' => [
+    'amqp' => [
+        'host' => 'rabbitmq.example.com',
+        'port' => '5672',
+        'user' => 'magento',
+        'password' => 'magento',
+        'virtualhost' => '/',
+        'ssl' => false
+    ]
+]
+```
+
+**For ActiveMQ Artemis (STOMP) - Available in 2.4.6+ (introduced in 2.4.9, backported to 2.4.6-2.4.8):**
+```php
+'queue' => [
+    'stomp' => [
+        'host' => 'activemq.example.com',
+        'port' => '61613',
+        'user' => 'artemis',
+        'password' => 'artemis',
+        'ssl' => false
+    ]
+]
+```
+
+**For Both Brokers - Available in 2.4.6+ (requires `default_connection`):**
+```php
+'queue' => [
+    'default_connection' => 'amqp', // Required when multiple brokers are configured
+    'amqp' => [
+        'host' => 'rabbitmq.example.com',
+        'port' => '5672',
+        'user' => 'magento',
+        'password' => 'magento',
+        'virtualhost' => '/',
+        'ssl' => false
+    ],
+    'stomp' => [
+        'host' => 'activemq.example.com',
+        'port' => '61613',
+        'user' => 'artemis',
+        'password' => 'artemis',
+        'ssl' => false
+    ]
+]
+```
+
+>[!NOTE]
```

Comment:
This is ExL note syntax. Unfortunately, it isn't supported in the devdocs, but I appreciate the attempt 😉.

For notes in devdocs, you can use the `<InlineAlert>` block described in the [devdocs theme](https://github.com/adobe/aio-theme/?tab=readme-ov-file#inlinealert-block-updated-2022-06-08). There's also an example of an `<InlineAlert>` block on line 204 below.